aws/resource_aws_iam_group_membership_test.go

```diff
@@ -242,3 +324,14 @@ resource "aws_iam_user" "user" {
 }
 `, groupName, membershipName, userNamePrefix)
 }
+
+func testAccAWSGroupMemberAddUnmanagedUser(groupName string, userName string) func() {
```

Comment:
We should probably create a complimentary `testAccAWSGroupMemberRemoveManagedUser`